packages/tokens/src/Spacing.tsx

```diff
@@ -0,0 +1,9 @@
+export const spacing = {
```

Comment:
Nit, lower case file name. Also should this be a default export?

Reply:
yes makes sense to me